### PR TITLE
README Updates

### PR DESCRIPTION
**Pull request by fugue-chris**
Added rules library and changed some of the upfront language.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,8 @@
 -   [Regula rules](#regula-rules)
     -   [Simple rules](#simple-rules)
     -   [Advanced rules](#advanced-rules)
--   [Control mapping](#control-mapping)
+    -   [Rule library](rule-library)
```
Comment (becki-at-luminal) [inline README.md]:
```suggestion
    -   [Rule library](#rule-library)
```

File: README.md [hunk 2]
```diff
@@ -132,11 +133,25 @@ The `fugue` API consists of four functions:
 -   `fugue.missing_resource(resource_type)` marks a resource as **missing**.
     This is useful if you for example _require_ a log group to be present.
 
-## Control mapping
+### Rule library
```
Comment (becki-at-luminal) [inline README.md]: Should we note which ones are proper rules and which are examples, so people know which ones to pick and choose?

File: README.md [hunk 3]
```diff
@@ -132,11 +133,25 @@ The `fugue` API consists of four functions:
...
+| AWS      | CloudTrail | cloudtrail\_log\_file\_validation\_enabled | CloudTrail log file validation should be enabled                                                           |
+| AWS      | KMS        | kms\_rotate                             | KMS CMK rotation should be enabled                                                                         |
+| AWS      | EBS        | ebs\_volume\_encrypted                   | EBS volume encryption should be enabled
+
+## Compliance controls
 
 ```ruby
 In Regula, _rules_ provide the lower-level implementation details, and
-_controls_ are policies that map to sets of rules.  Controls can
+_controls_ are compliance controls (e.g. CIS AWS Foundations Framework 4-1) that map to sets of rules.  Controls can
```
Comment (becki-at-luminal) [inline README.md]: Add comma after `e.g.` and change name: `e.g., CIS AWS Foundations Benchmark 4-1`

File: README.md [hunk 4]
```diff
@@ -16,19 +17,19 @@
 
 ## Introduction
 
-Regula is a very light (less than 100 lines of bash) framework around [opa] and
-[terraform] that allows you to more easily implement pre-flight policy checks,
-by providing a rigid structure for rules.
+Regula is a tool that evaluates Terraform infrastructure-as-code for potential security misconfigurations and compliance violations prior to deployment.
```
Comment (becki-at-luminal) [inline README.md]: @fugue-chris Should we mention Fugue anywhere here?

Comment (fugue-chris) [reply README.md]: No, I intentionally left Fugue out of it

File: README.md (hunk 2 above)
Comment (becki-at-luminal) [reply README.md]: Also, should we add links to the files?

Comment (fugue-chris) [reply README.md]: I actually didn't include any of the examples here. Perhaps we add another subsection on examples?

File: README.md [hunk 5]
```diff
@@ -16,19 +17,19 @@
...
+
+Regula includes a library of rules written in Rego, the policy language used by the Open Policy Agent ([opa]) project. Regula works with your favorite CI/CD tools such as Jenkins, Circle CI, and AWS CodePipeline; we’ve included a GitHub Actions example so you can get started quickly. Where relevant, we’ve mapped Regula policies to the CIS AWS Foundations Benchmark so you can assess your compliance posture.
```
Comment (becki-at-luminal) [inline README.md]: Maybe add links to the GitHub Actions repo and the example repo?

File: README.md (hunk 2 above)
Comment (becki-at-luminal) [reply README.md]: Oh right -- sorry! That would be nice. Or at least a sentence mentioning the folder and why the rules there aren't included with the rest. The directory structure is listed further down so we don't need to go into much detail, I think.

**Pull request merged**